models/examples/primary-care/acute/no-nominated-pharmacy/medical-prescriber/author/gmc/responsible-party/din/1-Prepare-Request-200_OK.json

```diff
@@ -0,0 +1,482 @@
+            "postalCode": "LS1 6AE"
+          }
+        ],
+        "generalPractitioner": [
+          {
+            "identifier": {
+              "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+              "value": "C81010"
+            }
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:56166769-c1c4-4d07-afa8-132b5dfca666",
+      "resource": {
+        "resourceType": "PractitionerRole",
+        "identifier": [
+          {
+            "system": "https://fhir.nhs.uk/Id/sds-role-profile-id",
+            "value": "200102238987"
+          }
+        ],
+        "practitioner": {
+          "reference": "urn:uuid:a8c85454-f8cb-498d-9629-78e2cb5fa47a"
+        },
+        "organization": {
+          "reference": "urn:uuid:3b4b03a5-52ba-4ba6-9b82-70350aa109d8"
+        },
+        "code": [
+          {
+            "coding": [
+              {
+                "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-SDSJobRoleName",
+                "code": "R8000",
+                "display": "Clinical Practitioner Access Role"
+              }
+            ]
+          }
+        ],
+        "telecom": [
+          {
+            "system": "phone",
+            "value": "01234567890",
+            "use": "work"
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:a8c85454-f8cb-498d-9629-78e2cb5fa47a",
+      "resource": {
+        "resourceType": "Practitioner",
+        "identifier": [
+          {
+            "system": "https://fhir.nhs.uk/Id/sds-user-id",
+            "value": "555086689106"
+          },
+          {
+            "system": "https://fhir.hl7.org.uk/Id/gmc-number",
+            "value": "6095103"
+          },
+          {
+            "system": "https://fhir.hl7.org.uk/Id/din-number",
+            "value": "977677"
+          }
+        ],
+        "name": [
+          {
+            "family": "BOIN",
+            "given": [
+              "C"
+            ],
+            "prefix": [
+              "DR"
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:3b4b03a5-52ba-4ba6-9b82-70350aa109d8",
+      "resource": {
+        "resourceType": "Organization",
+        "identifier": [
+          {
+            "system": "https://fhir.nhs.uk/Id/ods-organization-code",
+            "value": "A83008"
+          }
+        ],
+        "type": [
+          {
+            "coding": [
+              {
+                "system": "https://fhir.nhs.uk/CodeSystem/organisation-role",
+                "code": "RO76",
+                "display": "GP PRACTICE"
+              }
+            ]
+          }
+        ],
+        "name": "HALLGARTH SURGERY",
+        "address": [
+          {
+            "use": "work",
+            "type": "both",
+            "line": [
+              "HALLGARTH SURGERY",
+              "CHEAPSIDE",
+              "SHILDON",
+              "COUNTY DURHAM"
+            ],
+            "city": "SHILDON",
```

Comment:
Maybe remove the last two entries in `line` (or remove `city` & `district`). At the moment, these last two lines of the address are duplicated in the HL7 V3 message